substrate/utils/frame/remote-externalities/src/lib.rs

```diff
@@ -644,7 +741,7 @@ where
 			},
 		};
 		bar.finish_with_message("✅ Downloaded key values");
-		print!("\n");
+		println!();
```

Comment:
You sure?

Reply:
They work the same. I changed it only to reduce the amount of clippy warnings which were clustering the screen. :))